frontend/public/components/utils/service-level.tsx

```diff
@@ -271,14 +271,19 @@ export const useServiceLevelTitle = (): string => {
   return t('public~Service Level Agreement (SLA)');
 };
 
+export const useShowServiceLevelNotifications = (clusterID: string): boolean => {
+  const { level } = useGetServiceLevel(clusterID);
+  return level && (level === 'Eval' || level === 'None');
+};
+
 export const ServiceLevelNotification: React.FC<{
   clusterID?: string;
   toggleNotificationDrawer: () => void;
 }> = ({ clusterID, toggleNotificationDrawer }) => {
   const { t } = useTranslation();
   const { level, daysRemaining, trialDateEnd } = useGetServiceLevel(clusterID);
 
-  if (!level || (level !== 'Eval' && level !== 'None')) {
+  if (!useShowServiceLevelNotifications(clusterID)) {
```

Comment:
I would separate the hooks from the logic here as well for the same reason as above.

```ts
  const { level, daysRemaining, trialDateEnd } = useGetServiceLevel(clusterID);
  const showServiceLevelNotifications = useShowServiceLevelNotifications(clusterID);

  if (!showServiceLevelNotifications) {
    return null;
  }
```

What you have is fine, but to me pulling the hook call outside of the conditional make it more obvious that it's a hook that always needs to be called.

Reply:
Should be fixed now.